modoboa/core/api/v2/views.py

```diff
@@ -1,18 +1,30 @@
 """Core API v2 views."""
 
 import logging
+import oath
 
 from django.utils.html import escape
 from django.utils.translation import ugettext as _
-
-from django.contrib.auth import login
+from django.contrib.auth import login, get_user_model, password_validation
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.core.exceptions import ValidationError
+from django.views.decorators.cache import never_cache
+from django.utils.decorators import method_decorator
+from django.utils.http import urlsafe_base64_decode
+from django.db.models import Q
```

Comment:
I generally separate django and django.contrib imports (django imports first) and I sort them alphabetically.